README.md

```diff
@@ -57,7 +84,10 @@ The prerequisites are :
 The most basic way to start eXo Platform Community edition for *evaluation* purpose is to execute
 
 ```bash
-docker run -v exo_data:/srv/exo -p 8080:8080 exoplatform/exo-community
+docker network create -d bridge exo-network
+docker run -v mongo_data:/data/db -p 27017:27017 --name mongo --network=exo-network mongo:4.4
+docker run -e ES_JAVA_OPTS="-Xms2048m -Xmx2048m" -e node.name=exo -e cluster.name=exo -e cluster.initial_master_nodes=exo -e network.host=_site_ -v search_data:/usr/share/elasticsearch/data --name es --network=exo-network exoplatform/elasticsearch:2.0.3
+docker run -v exo_data:/srv/exo -p 8080:8080 -e EXO_ES_HOST=es --name exo --network=exo-network exoplatform/exo-community:6.3
```

Comment:
I think it will be helpful to state that those commands will create and run containers, and that once the contaienrs are created successfully, they could be started with
`` docker start $CONTAINER_NAME ``

Reply:
added